rpcs3/rpcs3qt/debugger_frame.cpp

```diff
@@ -302,13 +306,16 @@ void debugger_frame::keyPressEvent(QKeyEvent* event)
 				return;
 			}
 
-			static_cast<spu_thread*>(cpu.get())->debugger_float_mode ^= 1; // Switch mode
+			static_cast<spu_thread*>(cpu)->debugger_float_mode ^= 1; // Switch mode
 			return;
 		}
 		case Qt::Key_R:
 		{
-			register_editor_dialog* dlg = new register_editor_dialog(this, cpu, m_disasm.get());
-			dlg->show();
+			if (m_cpu)
```

Comment:
this changes the cpu object to m_cpu. was this intended or is this copy pasta from renaming cpu to m_cpu?